File: Per.java
import java.util.*;
public  class sample {

    //Formula of permutation nPr = n!/(n-r)!
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int r = sc.nextInt();
        int ans = fact(n) / fact(n - r);
        System.out.println(ans);
       
    }
    static int fact(int n) {
        if (n == 0 || n == 1) {
            return 1;
        }
        return n * fact(n - 1);
    }
}

File: per.py
n=int(input("Enter a number: "))
r=int(input("Enter r value: "))
def fact (n):
    if n == 0 or n == 1:
        return 1
    else:
        return n * fact(n - 1)
result = fact(n) // fact(n - r)
print("Permutation is:", result)